editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -61,31 +56,46 @@ const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
         maxHeight: "fit-content",
       }}
     >
-      <TabContext value={activeTab}>
+      <TabContext value={activeIndex.toString()}>
```

Comment:
I feel like I'm slightly missing something here!

MUI allows `TabContext` and `Tab` to take `string | number` for `value`, but `TabPanel` only accepts `string`?

What's happening here is we're using the `activeIndex`, as a string, to control the tab state instead of managing two variables.

Reply:
Hit this as well with initial tab implementation - it's a strange one - managing one variable definitely feels cleaner tho :ok_hand: